airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/format/consts.py

```diff
@@ -62,7 +67,7 @@ class Formatter(Enum):
 WARM_UP_INCLUSIONS = {
     Formatter.JAVA: [
         "spotless-maven-pom.xml",
-        "tools/gradle/codestyle/java-google-style.xml",
+        "tools/",  # Whole directory instead of just 'java-google-style.xml' to support airbyte-enterprise.
```

Comment:
I'm not sure why this change is required. And I think it could impact performance as we mount more things during the "warm up" phase.

Reply:
In airbyte-enterprise we have a symlink to `tools` and including a file inside of that only works if the symlink works.

I understand why you'd be worried by `tools` only contains 1.2 MB of files which basically never change. I think it's OK.